src/cbapi/psc/models/deviceInfo.yaml

```diff
@@ -0,0 +1,221 @@
+type: object
```

Comment:
This is the old device schema and I would recommend just device.yaml

Reply:
I just copied it, I'm not changing it for now.